### PR TITLE
Separate followups and registrations

### PR DESCRIPTION
**Pull request by harimohanraj89**
**Story card:** TBD

## Because

It is confusing to have assigned follow-up info right next to registration info in the same table.

## This addresses

* Separate followups and registrations into two separate tables in:
  * District analytics report
  * District new report - Details page

While we're in here, we make a few other enhancements:
* Allow more than one sortable table on a single page
* Consistent-ize our month format across the board to `Mon-YYYY`
* Consistent-ize our quarter format across the board to `QX YYYY`
* Fix a table sorting bug on the Missed Visits page - Total column

### Before

Screen Shot 2020-09-25 at 5 52 39 PM
Screen Shot 2020-09-25 at 5 53 01 PM

### After

Screen Shot 2020-09-25 at 5 52 17 PM
Screen Shot 2020-09-25 at 5 52 31 PM

### CODE REVIEW

File: app/helpers/quarter_helper.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def quarter_string(date)
     year = date.year
     quarter = quarter(date)
 
-    "#{year} Q#{quarter}"
+    "Q#{quarter}-#{year}"
```
Comment (claudiovallejo) [inline app/helpers/quarter_helper.rb:35]: @harimohanraj89 added the hyphen to the quarter string. Did a quick check, and don't think this breaks any tests.

File: app/views/analytics/districts/_follow_ups_table.html.erb [hunk 2]
```diff
@@ -1,50 +1,51 @@
...
-        <th class="row-label sort-label sort-label-small" style="text-align: left;" data-sort-default>Name</th>
-        <!--Total assigned patients-->
-        <th class="row-label sort-label" data-sort-default data-sort-method="number">Total</th>
+        <th class="row-label sort-label" style="text-align: left;" data-sort-default>Facilities</th>
 
         <!--Patients with BP taken-->
-        <% dates_for_periods(period, 6, include_current_period: @show_current_period).each do |date| %>
-          <th class="row-label sort-label sort-label-small" data-sort-method="number">
+        <% dates_for_periods(period, 6, include_current_period: @show_current_period).each_with_index do |date, index| %>
+          <th class="row-label sort-label" <%= "data-sort-default" if index == 5 %> data-sort-method="number">
```
Comment (prabhanshuguptagit) [inline app/views/analytics/districts/_follow_ups_table.html.erb:42]: [nit] How about
```
if date == dates.last
```

**Review by prabhanshuguptagit — CHANGES_REQUESTED** [review]

**Review by claudiovallejo — APPROVED** [review]

File: app/views/analytics/districts/_follow_ups_table.html.erb (hunk 2 above)
Comment (harimohanraj89) [reply app/views/analytics/districts/_follow_ups_table.html.erb:42]: Love it.

File: app/views/analytics/districts/_registrations_table.html.erb [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<div class="card mt-0 pr-0 pr-md-3">
```
[the remark this answers is not in the document]
Comment (harimohanraj89) [reply app/views/analytics/districts/_registrations_table.html.erb:1]: We considered it :) We're gonna wait until someone asks for it.

**Review by prabhanshuguptagit — APPROVED** [review]
✨

**Pull request merged**